### PR TITLE
ts-web: catching up with sdk changes

### PR DESCRIPTION
**Pull request by pro-wh**
also a client-rt release

### CODE REVIEW

File: client-sdk/ts-web/core/reflect-go/go.mod [hunk 1]
```diff
@@ -3,59 +3,3 @@ module github.com/oasisprotocol/oasis-sdk/client-sdk/ts-web/core/reflect-go
 go 1.17
 
 require github.com/oasisprotocol/oasis-core/go v0.2103.6
-
-require (
```
Comment (kostko) [inline client-sdk/ts-web/core/reflect-go/go.mod:7]: How did this happen? This is a new thing with Go 1.17 so it should remain.

Comment (pro-wh) [reply client-sdk/ts-web/core/reflect-go/go.mod:7]: oh I'm on the wrong version of Go locally :facepalm:

Comment (pro-wh) [reply client-sdk/ts-web/core/reflect-go/go.mod]: removing this commit

**Comment by pro-wh** [comment]
gonna hold off on updating core. maybe we can first sort out some grpc-web error handling improvements

**Review by kostko — APPROVED** [review]

**Pull request merged**